src/kvstore/NebulaStore.cpp

```diff
@@ -359,10 +359,12 @@ ResultCode NebulaStore::get(GraphSpaceID spaceId,
 }
 
 
-ResultCode NebulaStore::multiGet(GraphSpaceID spaceId,
-                                 PartitionID partId,
-                                 const std::vector<std::string>& keys,
-                                 std::vector<std::string>* values) {
+ErrorOr<ResultCode, std::vector<Status>> NebulaStore::multiGet(
+        GraphSpaceID spaceId,
+        PartitionID partId,
+        const std::vector<std::string>& keys,
+        std::vector<std::string>* values,
+        bool returnPartly) {
```

Comment:
Do we really need the param here? 
If only partial results returned, we could return ResultCode::PARTIAL_RESULTS;

Reply:
I think we'd better keep it, if we return `ResultCode::PARTIAL_RESULTS` we can't tell the difference between `key not exists` and `value is empty`,